### PR TITLE
feat(redux): add defaultCulture and defaultSubfolder keys on locale

### PR DESCRIPTION
**Pull request by vinicaval**
## Description

- created getCountryCultures

BREAKING CHANGES:
- renamed getCountryCultureCode to getCountryCulture
- changed types
- removed key structure from serverInitialState
- renamed exports to 'content*'. An example: actionTypesContent to contentActionTypes
- renamed exports to 'locale*'. An example: middlewaresLocale to localeMiddlewares

## Checklist

- [ ] The commit message follows our guidelines
- [ ] Tests for the respective changes have been added
- [ ] The code is commented, particularly in hard-to-understand areas
- [ ] The labels and/or milestones were added

## Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

File: packages/redux/src/locale/serverInitialState.ts [hunk 1]
```diff
@@ -39,7 +45,9 @@ export default ({ model }: { model: Model }): ServerInitialState => {
       ],
       newsletterSubscriptionOptionDefault,
       platformId: countryId,
-      structure: subfolder,
+      // requestSourceCountryCode,
```
Comment (joaoprcosta) [inline packages/redux/src/locale/serverInitialState.ts:48]: Missing sourceCountryCode

File: tests/__fixtures__/locale/locale.fixtures.ts [hunk 2]
```diff
@@ -77,4 +78,7 @@ export const mockModel = {
   countryId: 216,
   cultureCode: 'en-US',
   newsletterSubscriptionOptionDefault: false,
+  defaultCulture: 'en-US',
+  defaultSubfolder: '/en-us',
+  requestSourceCountryCode: 'PT',
```
Comment (joaoprcosta) [inline tests/__fixtures__/locale/locale.fixtures.ts:83]: Name the key as 'sourceCountryCode'.

**Review by boliveira — APPROVED** [review]

File: packages/redux/src/types/model.types.ts [hunk 3]
```diff
@@ -13,6 +13,9 @@ type Common = {
   currencyCultureCode: string;
   designers: Designers;
   newsletterSubscriptionOptionDefault: boolean;
+  // requestSourceCountryCode: string;
```
[the remark this answers is not in the document]
Comment (vinicaval) [reply packages/redux/src/types/model.types.ts]: done

File: packages/redux/src/locale/serverInitialState.ts (hunk 1 above)
Comment (vinicaval) [reply packages/redux/src/locale/serverInitialState.ts]: done

File: tests/__fixtures__/locale/locale.fixtures.ts (hunk 2 above)
Comment (vinicaval) [reply tests/__fixtures__/locale/locale.fixtures.ts:91]: the key on "first render" is 'requestSourceCountryCode' after the serverInitialState it becomes 'sourceCountryCode' as you said. Can I leave it like this?

Comment (joaoprcosta) [reply tests/__fixtures__/locale/locale.fixtures.ts:91]: sourceCountryCode: 'PT',

**Pull request merged**